app/views/ui/register.html.haml

```diff
@@ -17,21 +17,21 @@
             %label.control-label.col-sm-2 Full Name
             .col-sm-6
               %input.form-control(type="text")
-        %fieldset.credit_card
-          .form-group
-            %label.control-label.col-sm-2 Credit Card Number
-            .col-sm-6
-              %input.form-control(type="text")
-          .form-group
-            %label.control-label.col-sm-2 Security Code
-            .col-sm-6
-              %input.form-control(type="text")
-          .form-group
-            %label.control-label.col-sm-2 Expiration
-            .col-sm-3
-              = select_month(Date.today, {add_month_numbers: true},  class: 'form-control')
-            .col-sm-2
-              = select_year(Date.today.year, {start_year: Date.today.year, end_year: Date.today.year + 4}, class: 'form-control')
+        -#%fieldset.credit_card
```

Comment:
I am not really positive on this block of code being commented. You have it in UI views, otherwise it only distracts now.